app/javascript/components/assessment/AssessmentTable.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import _ from 'lodash';
 import { PropTypes } from 'prop-types';
+import { Button, Card, Col, Dropdown, Form, InputGroup, OverlayTrigger, Row, Tooltip } from 'react-bootstrap';
+
```

Comment:
Only added `import { formatTimestamp } from '../../lib/DateTimeUtils';`. Everything else here is just moving `imports` around / alphabetizing them.